src/main/java/org/aludratest/config/impl/SimplePreferences.java

```diff
@@ -29,11 +29,16 @@
  */
 public class SimplePreferences extends AbstractPreferences implements MutablePreferences {
 
-    private Map<String, String> values = new HashMap<String, String>();
+    private Map<String, String> values;
```

Comment:
Why this change? Is this a SonarQube recommendation? The compiler will generate exactly that constructor you added manually...